package.json

```diff
@@ -46,74 +46,72 @@
+    "fix-format-tools:js": "npx prettier --config .prettierrc -w tools/ci tools/docs .prettierrc .eslintrc.cjs .puppeteerrc.cjs .github",
+    "fix-format-tools:py": "./tools/ci/run/format-black-py.sh tools",
+    "fix:lint": "npm-run-all fix-lint:*",
+    "fix-lint:tools": "npm-run-all fix-lint-tools:*",
+    "fix-lint-tools:js": "npx eslint --fix --ext .js,.cjs,.mjs --config .eslintrc.cjs tools/ci tools/docs .eslintrc.cjs .puppeteerrc.cjs",
+    "fix-lint:src": "npm-run-all fix-lint-src:*",
+    "fix-lint-src:js": "npx eslint --fix --ext .js,.cjs,.mjs --config .eslintrc.cjs src test",
+    "fix-lint:docs": "npm-run-all fix-lint-docs:*",
+    "fix-lint-docs:js": "npx eslint --fix --ext .js,.cjs,.mjs --config .eslintrc.cjs docs",
+    "type": "npm-run-all type:*",
+    "type:tools": "npm-run-all type-tools:*",
+    "type-tools:py": "./tools/ci/run/type-mypy-py.sh tools",
     "test": "node test/integration/test.cjs",
-    "ci-src": "npm-run-all format-src lint-src test-unit-src test-unit-test test",
-    "ci-docs": "npm-run-all format-docs lint-docs",
-    "ci-tools": "npm-run-all format-tools lint-tools type-tools",
-    "ci": "npm-run-all ci-src ci-docs ci-tools",
-    "docs-gen-thumbnail": "./tools/ci/run/docs-gen-thumbnail.sh",
-    "docs-gen-thumbnail-gsutil": "./tools/ci/run/docs-gen-thumbnail-gsutil.sh",
-    "docs-build": "./tools/ci/run/docs-build.sh",
-    "docs-deploy": "./tools/ci/run/docs-deploy.sh",
-    "pkg-build-desktop": "./tools/ci/run/pkg-build-desktop.sh",
-    "pkg-build-desktop-clangformat": "./tools/ci/run/pkg-build-desktop-clangformat.sh",
-    "pkg-build-desktop-clangtidy": "./tools/ci/run/pkg-build-desktop-clangtidy.sh",
-    "pkg-build-wasm": "./tools/ci/run/pkg-build-wasm.sh",
-    "pkg-build-wasm-wocpp": "./tools/ci/run/pkg-build-wasm-wocpp.sh",
-    "pkg-build-wasm-wocpp-gsutil": "./tools/ci/run/pkg-build-wasm-wocpp-gsutil.sh",
-    "pkg-build-ts": "./tools/ci/run/pkg-build-ts.sh",
-    "pkg-rollup-js": "./tools/ci/run/pkg-rollup-js.sh",
-    "pkg-set-version-js": "./tools/ci/run/pkg-set-version-js.sh",
-    "pkg-build-js": "./tools/ci/run/pkg-build-js.sh",
-    "pkg-purge-js": "./tools/ci/run/pkg-purge-js.sh"
+    "test:man": "node test/integration/man.cjs",
+    "test:unit-test": "npx jest --config=./test/integration/modules/jest.config.cjs --verbose",
+    "test:unit-src": "NODE_OPTIONS='--experimental-vm-modules' npx jest --config test/unit/jest.config.js --verbose",
+    "ci": "npm-run-all ci:*",
+    "ci:src": "npm-run-all format:src lint:src test:unit-src test:unit-test test",
+    "ci:docs": "npm-run-all format:docs lint:docs",
+    "ci:tools": "npm-run-all format:tools lint:tools type:tools",
+    "docs:build": "./tools/ci/run/docs-build.sh",
+    "docs:deploy": "./tools/ci/run/docs-deploy.sh",
+    "docs:gen-thumbnail": "./tools/ci/run/docs-gen-thumbnail.sh",
+    "docs:gen-thumbnail-gsutil": "./tools/ci/run/docs-gen-thumbnail-gsutil.sh",
+    "pkg:rollup-js": "./tools/ci/run/pkg-rollup-js.sh",
```

Comment:
It would be good to have these pkg scripts without pkg as well:
```
"rollup-js": "./tools/ci/run/pkg-rollup-js.sh",
"pkg:rollup-js": "npm-run-all rollup-js",
```